pygeos/_geos.pxd

```diff
@@ -8,30 +8,40 @@ Example:
 """
 
 cdef extern from "geos_c.h":
+    # Types
     ctypedef void *GEOSContextHandle_t
+    ctypedef struct GEOSGeometry
+    ctypedef struct GEOSCoordSequence
+    ctypedef void (*GEOSMessageHandler_r)(const char *message, void *userdata)
 
+    # GEOS Context & Messaging
     GEOSContextHandle_t GEOS_init_r() nogil
     void GEOS_finish_r(GEOSContextHandle_t handle) nogil
+    void GEOSContext_setErrorMessageHandler_r(GEOSContextHandle_t handle, GEOSMessageHandler_r ef, void* userData) nogil
```

Comment:
See here on why I removed the `except` on each line:

https://cython.readthedocs.io/en/latest/src/userguide/language_basics.html#checking-return-values-of-non-cython-functions